config/initializers/constants.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Gitscm
+  TAGLINES = %w[
+    fast-version-control
+    everything-is-local
+    distributed-even-if-your-workflow-isnt
+    local-branching-on-the-cheap
+    distributed-is-the-new-centralized
+  ].freeze
+
+  VIDEOS = [
+    [1, "41027679", "Git Basics", "What is Version Control?", "what-is-version-control", "05:59"],
+    [2, "41381741", "Git Basics", "What is Git?", "what-is-git", "08:15"],
+    [3, "41493906", "Git Basics", "Get Going with Git", "get-going", "04:26"],
+    [4, "41516942", "Git Basics", "Quick Wins with Git", "quick-wins", "05:06"],
+  ].freeze
+
+  REDIRECT = {
```

Comment:
It seems that these redirects are refering to the first edition of Progit. So they should be erased with the corresponding code as we're phasing out Progit 1rst Ed.